lnrpc/invoicesrpc/addinvoice.go

```diff
@@ -504,13 +529,13 @@ func chanCanBeHopHint(channel *channeldb.OpenChannel, cfg *AddInvoiceConfig) (
 	if !isRemoteNodePublic {
 		log.Debugf("Skipping channel %v due to "+
 			"counterparty %x being unadvertised",
-			chanPoint, remotePub)
+			chanPoint, channel.RemotePubkey)
```

Comment:
Shouldn't this be `remotePub` or `.SerializeCompressed()` to have the bytes printed in base16?

Reply:
Ah yes, good catch! Those woulda been some ugly logs.